From 04cdbacc26c7e38d3bd684235b51c79ab64b6026 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Tue, 8 Nov 2022 11:07:02 +0100
Subject: [PATCH] man: add note that network-generator is not a generator

Also fix indentation.

(cherry picked from commit 2fa6574e835566c2aa5cbf4167ecee316f71bf98)

Related #2138081
---
 man/systemd-network-generator.service.xml | 11 +++++++++--
 1 file changed, 9 insertions(+), 2 deletions(-)

diff --git a/man/systemd-network-generator.service.xml b/man/systemd-network-generator.service.xml
index 6b7e2564d4..2ddeadfc54 100644
--- a/man/systemd-network-generator.service.xml
+++ b/man/systemd-network-generator.service.xml
@@ -41,10 +41,17 @@
     </para>
 
     <para>Files are generated in <filename>/run/systemd/network/</filename>.</para>
+
+    <para>Note: despite the name, this generator executes as a normal systemd service and is
+    <emphasis>not</emphasis> an implementation of the
+    <citerefentry><refentrytitle>systemd.generator</refentrytitle><manvolnum>7</manvolnum></citerefentry>
+    concept.</para>
   </refsect1>
 
-  <refsect1><title>Kernel command line options</title>
-  <para>This tool understands the following options:</para>
+  <refsect1>
+    <title>Kernel command line options</title>
+
+    <para>This tool understands the following options:</para>
 
     <variablelist class='kernel-commandline-options'>
       <varlistentry>
